### ISSUE TITLE
website has errors on annotations accessor

### ISSUE THREAD

**Issue by erg** [slavapestov/factor#1427]
This is in the logs quite a bit:

```
[2015-08-08T02:45:57.264985Z] NOTICE accepted-connection: remote: 79.170.44.85:44831, local: 0.0.0.0:8080
[2015-08-08T02:45:57.266864Z] NOTICE httpd-hit: { "GET" URL" http://docs.factorcode.org/wp-admin/" }
[2015-08-08T02:45:57.267174Z] NOTICE httpd-header: { "user-agent" f }
[2015-08-08T02:45:57.267404Z] NOTICE httpd-header: { "x-forwarded-for" f }
[2015-08-08T02:45:57.291069Z] NOTICE httpd-hit: { "GET" URL" http://paste.factorcode.org/paste?id=3346" }
[2015-08-08T02:45:57.291596Z] NOTICE httpd-header: { "user-agent" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36" }
[2015-08-08T02:45:57.291832Z] NOTICE httpd-header: { "x-forwarded-for" f }
[2015-08-08T02:45:57.300084Z] DEBUG init-user: f
[2015-08-08T02:45:57.322821Z] ERROR do-request: Generic word annotations<< does not define a method for the POSTPONE: f class.
[--------------------] ERROR do-request: Dispatching on object: f```

**Comment by erg** [comment]
3386 is a deleted paste.

**Comment by mrjbq7** [comment]
Yep, looks like this line:

https://github.com/slavapestov/factor/blob/master/extra/webapps/pastebin/pastebin.factor#L86

has the issue of ``<paste-state> select-tuple`` returning ``f``...

**Comment by mrjbq7** [comment]
Fixed in 67ea8e7fa06da6094effdb4358dd340f5d8b100e.
